src/Framework/AspNetCoreAnalyzers/test/Microsoft.AspNetCore.App.Analyzers.Test.csproj

```diff
@@ -14,11 +14,20 @@
     <!-- Also bring in Microsoft.AspNetCore.App.Analyzers. -->
     <ProjectReference Include="..\src\CodeFixes\Microsoft.AspNetCore.App.CodeFixes.csproj" />
 
+    <!--
+    <ProjectReference Include="..\src\Analyzers\Microsoft.AspNetCore.App.Analyzers.csproj"
```

Comment:
Remove these?

I wonder if there is a way for us to set up a sample project in this directory with the references hooked up so that they play nice with our build system. I recall you did something similar when introducing the routing analyzers?

Reply:
I did this with the routing analyzers. I want to keep the commented out project references here, but I'll add a comment explaining what they're for.